app/lib/import_export.rb

```diff
@@ -72,7 +72,8 @@ module ImportExport # rubocop:todo Metrics/ModuleLength
                                 healthcare_personnel healthcare_personnel_facility_name crew_on_passenger_or_cargo_flight member_of_a_common_exposure_cohort
                                 member_of_a_common_exposure_cohort_type exposure_risk_assessment monitoring_plan exposure_notes full_status symptom_onset
                                 case_status lab_1_type lab_1_specimen_collection lab_1_report lab_1_result lab_2_type lab_2_specimen_collection lab_2_report
-                                lab_2_result jurisdiction_path assigned_user gender_identity sexual_orientation].freeze
+                                lab_2_result jurisdiction_path assigned_user gender_identity sexual_orientation race_other race_unknown
```

Comment:
So unfortunately this isn't sufficient for making sure these new columns are appended to the exported. On [line 103](https://github.com/SaraAlert/SaraAlert/blob/master/app/lib/import_export.rb#L103) we append `extended_isolation` specifically which makes it show up last in the single monitoree download and all monitorees downloads. I'm not clear on/remembering why this was done with extended isolation, but it'll need to be resolved as we informed jurisdictions their scripts can expect these new columns to be appended at the end.
<img width="685" alt="Screen Shot 2021-02-22 at 4 24 50 PM" src="https://user-images.githubusercontent.com/11698457/108772337-e2423c80-752a-11eb-9cfa-c6784acc89fc.png">

Reply:
Would we be able to just add `extended_isolation_date` as a field in `SARA_ALERT_FORMAT_FIELDS` before these new fields? now that I'm thinking more about this, was `gender_identity` and `sexual_orientation` added after `extended_isolation_date` or the other way around? we probably ran into the same problem if the former was the case but either way, I think it would be good to just add `extended_isolation_field` as an official field to SAF instead of just appending it to the `FULL_HISTORY_PATIENTS_FIELDS` because we're definitely going to be continuing to add more fields in the future

Reply:
> Would we be able to just add extended_isolation_date as a field in `SARA_ALERT_FORMAT_FIELDS` before these new fields?

I don't see why not, but I feel like there must have been a reason it was added the way it is. Are there any cases you can think of where extended isolation shouldn't be included with the SARA_ALERT_FORMAT_FIELDS? We support it on import + export so I can't think of any myself.

>was gender_identity and sexual_orientation added after extended_isolation_date or the other way around?
Added before I believe, so yeah I think this was a problem then too.

Reply:
here's the PR thread about this: https://github.com/SaraAlert/SaraAlert/pull/336#discussion_r485103442

and here was our slack convo from 9/8: https://mitrecorp.slack.com/archives/D01695F4MF1/p1599589661028500

it sounded like we decided to implement it this way at the time because `extended_isolation` isn't a field that is imported, so we didn't want that to be a field in the Sara Alert Format. We can discuss with Kelly again, but I think this is going to be an ongoing issue where we'll either have to just add this field to SAF or continue to move this field to a different column every time we add new fields (which I think is less ideal)

Reply:
Nice investigation work - totally forgot about that whole conversation. Okay so for now it doesn't affect the import template (which was my original worry), but it does mean that these new race fields won't be the last column in the export specifically. Yeah this isn't great - one option is that `FULL_HISTORY_PATIENTS_FIELDS` and `FULL_HISTORY_PATIENTS_HEADERS` maybe shouldn't depend on `SARA_ALERT_FORMAT_FIELDS`. Thoughts?

Reply:
that's possible too, but I also think it's very likely that we'll continue to add more fields to `SARA_ALERT_FORMAT_FIELDS` in the future

Reply:
Agreed, and if we separate out `FULL_HISTORY_PATIENTS_FIELDS` as a separate field that's not dependent on `SARA_ALERT_FORMAT_FIELDS` then that wouldn't be a problem I don't think. The only issue with that is you'd have t update both rather than just one when a field is added to the Sara Alert Format

Reply:
Yeah I think that's another viable solution as well. My only concern with that is with users potentially making the assumption that the formats are the same (other than one of them including `id`), leading to them copying/pasting or writing scripts not being aware of this discrepancy. Should we ask Kelly and @holmesie for thoughts?

Reply:
What I'm suggesting wouldn't be any change for users, would entirely be on the code side. So if the concern is about there being a very slight difference between the Sara Alert Format export fields vs the Full Patient History export fields then yeah that's been a thing for awhile. So if you want to make sure folks are aware of this difference I don't think it hurts.

Reply:
My concern is the slight difference between the formats, I'll start a slack thread in the UI channel to discuss and get Kelly's input

Reply:
added comment regarding our decision to keep export fields as is here: https://github.com/SaraAlert/SaraAlert/pull/689/commits/5f5f17dfb7ad63738aca9d7b88ed076131222a5b